src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/workflows.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:framework="http://symfony.com/schema/dic/symfony"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
+                        http://symfony.com/schema/dic/symfony http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+    <framework:config>
+        <framework:workflow name="my_workflow">
+            <framework:marking-store type="multiple_state">
+                <framework:argument>a</framework:argument>
+                <framework:argument>a</framework:argument>
+            </framework:marking-store>
+            <framework:support>Symfony\Bundle\FrameworkBundle\Tests\DependencyInjection\FrameworkExtensionTest</framework:support>
+            <framework:place>first</framework:place>
+            <framework:place>last</framework:place>
+            <framework:transition name="foobar">
+                <framework:from>a</framework:from>
+                <framework:to>a</framework:to>
+            </framework:transition>
+        </framework:workflow>
+        <framework:workflow name="my_state_machine">
+            <framework:marking-store type="single_state" />
+            <framework:support>Symfony\Bundle\FrameworkBundle\Tests\DependencyInjection\FrameworkExtensionTest</framework:support>
+            <framework:place>first</framework:place>
+            <framework:place>last</framework:place>
+            <framework:transition name="foo">
+                <framework:from>a</framework:from>
+                <framework:to>a</framework:to>
```

Comment:
I added two more meaningful examples (one taken from @lyrixx' demo application and the other one from the docs written by @Nyholm). I also noticed that there was no way to configure the type of the workflow. This is fixed now. Tests have been expanded too.